packages/flutter_tools/lib/src/dart/language_version.dart

```diff
@@ -0,0 +1,67 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
2020

Reply:
For reasons(?), all copyright comments, including for new files, retain the original copyright date.